dags/hivemind_etl_helpers/src/db/telegram/extract/messages.py

```
@@ -0,0 +1,76 @@
+        ---------
+        tg_messages : list[TelegramMessagesModel]
+            the telegram messages
+        """
+        query = "MATCH (c:TGChat {id: $chat_id})<-[:SENT_IN]-(message:TGMessage)"
+
+        where_clause: str | None = None
+        from_date_timestamp = int(from_date.timestamp() * 1000)
+        if from_date:
+            where_clause = f"""
+            AND message.date >= $from_date_timestamp
+            """
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)